sig-TEMPLATE/README.md

```diff
@@ -0,0 +1,13 @@
+# SIG Template
```

Comment:
Include link to Charter in the template, for people only looking at the README?

Reply:
good catch! thank you Quentin 😁